MessageViewController/MessageView.swift

```diff
@@ -92,22 +114,60 @@ public final class MessageView: UIView, MessageTextViewListener {
         didSet { setNeedsLayout() }
     }
 
-    public func set(buttonIcon: UIImage?, for state: UIControlState) {
-        button.setImage(buttonIcon, for: state)
+    public func set(buttonIcon: UIImage?, for state: UIControlState, type: buttonType) {
+        switch type {
+            case .left:
+                setLeft(buttonIcon: buttonIcon, for: state)
+                break
+            case .right:
+                setRight(buttonIcon: buttonIcon, for: state)
+                break
+        }
         buttonLayoutDidChange()
     }
 
-    public func set(buttonTitle: String, for state: UIControlState) {
-        button.setTitle(buttonTitle, for: state)
+    public func set(buttonTitle: String, for state: UIControlState, type: buttonType) {
+        switch type {
+        case .left:
+            setLeft(buttonTitle: buttonTitle, for: state)
+            break
+        case .right:
+            setRight(buttonTitle: buttonTitle, for: state)
+            break
+        }
         buttonLayoutDidChange()
     }
 
-    public var buttonTint: UIColor {
-        get { return button.tintColor }
+    private func setLeft(buttonIcon: UIImage?, for state: UIControlState) {
+        leftButton.setImage(buttonIcon, for: state)
+    }
+
+    private func setLeft(buttonTitle: String, for state: UIControlState) {
+        leftButton.setTitle(buttonTitle, for: state)
+    }
+
+    private func setRight(buttonIcon: UIImage?, for state: UIControlState) {
+        rightButton.setImage(buttonIcon, for: state)
+    }
+
+    private func setRight(buttonTitle: String, for state: UIControlState) {
+        rightButton.setTitle(buttonTitle, for: state)
+    }
```

Comment:
Nit: These look like unnecessary wrappers for these methods. It would be better to just call them directly